### PR TITLE
test handle put_public_pmid

### PR DESCRIPTION
**Pull request by benjaminbollen**
[Review on Reviewable](https://reviewable.io/reviews/dirvine/routing/175)

### CODE REVIEW

File: src/routing_table.rs [hunk 1]
```diff
@@ -266,7 +266,7 @@ impl RoutingTable {
         if self.routing_table.len() < GROUP_SIZE {
             return true;
         }
-        let furthest_close_node = self.routing_table[GROUP_SIZE].clone();
+        let furthest_close_node = self.routing_table[GROUP_SIZE - 1].clone();
```
Comment (benjaminbollen) [inline src/routing_table.rs]: Found a bug I introduced in RoutingTable before; will write accompanying test for `address_in_our_close_group`

**Comment by benjaminbollen** [comment]
werid about the coverage result here:  the PR only adds a test; and removes a commented code block.  Probably the latter made coverage drop.

File: src/routing_node.rs [hunk 2]
```diff
@@ -677,18 +677,24 @@ impl<F> RoutingNode<F> where F: Interface {
         Ok(())
     }
 
+    /// Every node needs to republish its public key directly to PublicPmid.name every 10 minutes.
+    /// Sentinel will query this pool.  No handle_get_public_pmid is needed.
```
Comment (chandraprakash) [inline src/routing_node.rs]: Can you remove above comment. there is no api for PutPublicPmid...
This only happens while bootstrapping.

File: src/routing_table.rs (hunk 1 above)
Comment (chandraprakash) [inline src/routing_table.rs]: good catch !

**Pull request merged**